edge/cmd/edgecore/app/server.go

```diff
@@ -125,21 +124,6 @@ offering HTTP client capabilities to components of cloud to reach HTTP servers r
 				}
 				config.Modules.Edged.NodeIP = ip.String()
 				klog.Infof("Get IP address by custom interface successfully, %s: %s", config.Modules.Edged.CustomInterfaceName, config.Modules.Edged.NodeIP)
-			} else {
```

Comment:
edgestream is use the nodeIP to register the session to the cloudstream, if we just delete the code, the stream may have some probleam
```
header.Add(stream.SessionKeyInternalIP, e.nodeIP)
```
	internalIP := r.HeaderParameter(stream.SessionKeyInternalIP)
	if internalIP == "" {
		internalIP = strings.Split(r.Request.RemoteAddr, ":")[0]
	}
```

```